### PR TITLE
[CI:DOCS] Document --read-only --rootfs requirements

### PR DESCRIPTION
**Pull request by rhatdan**
Add entry to troubleshooting to document how to setup a read-only rootfs to
use with Podman.

Fixes: https://github.com/containers/podman/issues/5895

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: troubleshooting.md [hunk 1]
```diff
@@ -617,3 +617,29 @@ If you encounter a `fuse: device not found` error when running the container ima
...
 file to `/etc/modules.load.d`.  See `man modules-load.d` for more details.
+
+### 25) podman run --rootfs link/to//read/only/dir does not work
+
+https://github.com/containers/podman/issues/5895#
+
+#### Symptom
+
+Rootless Podman requires certain files to exist in a file system in order to run.
+Podman will create /etc/resolv.conf /etc/hosts and other fds on the rootfs in order
```
Comment (TomSweeneyRedHat) [inline troubleshooting.md:628]:
```suggestion
Podman will create /etc/resolv.conf, /etc/hosts and other file decriptors on the rootfs in order
```

File: troubleshooting.md [hunk 2]
```diff
@@ -617,3 +617,29 @@ If you encounter a `fuse: device not found` error when running the container ima
 the fuse kernel module has not been loaded on your host system.  Use the command `modprobe fuse` to load the
 module and then run the container image afterwards.  To enable this automatically at boot time, you can add a configuration
 file to `/etc/modules.load.d`.  See `man modules-load.d` for more details.
+
+### 25) podman run --rootfs link/to//read/only/dir does not work
+
+https://github.com/containers/podman/issues/5895#
```
Comment (TomSweeneyRedHat) [inline troubleshooting.md:623]: I'd prefer to see a short description here;
```suggestion
An error such as "OCI runtime error" on a read-only filesystem or the error "{image} is not an absolute path or is a symlink" are often times indicators for this issue.  For more details, review this [issue](
https://github.com/containers/podman/issues/5895).
```

**Comment by TomSweeneyRedHat** [comment]
LGTM
but gating is saying you didn't sign it?  Looks like you did to me.....

**Comment by rhatdan** [comment]
Updated and fixed some spelling issues.

**Review by TomSweeneyRedHat — COMMENTED** [review]
LGTM

**Pull request merged**